modules/ossm-customizing-istio-configuration.adoc

```diff
@@ -22,3 +22,7 @@ For a list of available configuration for the `values` field, refer to link:http
 * link:https://artifacthub.io/packages/helm/istio-official/gateway?modal=values[Gateway parameters]
 * link:https://artifacthub.io/packages/helm/istio-official/cni?modal=values[CNI parameters]
 * link:https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values[ZTunnel parameters]
+
```

Comment:
This comment is not related to this PR, but while looking at this doc, I noticed that in Line 15, we mention `Istio` instance name as `istio-sample`. AFAIR the name would be `default` (atleast I noticed this while using Sail Operator) @luksa @dgn do you know the correct name?

Reply:
I haven't checked the console, but I think you're right that the name is `default`, because the form is populated from [alm-examples](https://github.com/istio-ecosystem/sail-operator/blob/bd6e0482446decb56d55a7344296a29408dee4b4/bundle/manifests/sailoperator.clusterserviceversion.yaml#L11).

Reply:
@luksa @sridhargaddam , I can fix it right away in this PR. 👍